modules/user_management.js

```diff
@@ -74,6 +74,74 @@ class UserManagement {
         }]
       }
     })
+
+    // Add a single literature quiz point to a user.
+    dispatcher.command('addpoint', this._guild.id, (authorMember, tokenized) => {
+      util.requireRole(authorMember, this._managementRole)
+
+      const USAGE = `${this._config.get('commandPrefix') || 'p!'}addpoint @user`
+      util.requireParameterCount(tokenized, 1, USAGE)
+      util.requireParameterFormat(tokenized[0], arg => arg.startsWith('<@') && arg.endsWith('>'), USAGE)
+
+      const userId = tokenized[0].replace(/[<@!>]/g, '')
+
+      util.log(`addpoint to ${userId} by ${authorMember.id}`)
+      
+      // Call a helper method that conducts the increment and produces the output embed
+      return this._addLitQuizPoint(userId, userRepository, tokenized[0])
+    })
+
+    // Remove a single literature quiz point from a user.
+    dispatcher.command('delpoint', this._guild.id, (authorMember, tokenized) => {
+      util.requireRole(authorMember, this._managementRole)
+
+      const USAGE = `${this._config.get('commandPrefix') || 'p!'}delpoint @user`
+      util.requireParameterCount(tokenized, 1, USAGE)
+      util.requireParameterFormat(tokenized[0], arg => arg.startsWith('<@') && arg.endsWith('>'), USAGE)
+
+      const userId = tokenized[0].replace(/[<@!>]/g, '')
+
+      util.log(`delpoint from ${userId} by ${authorMember.id}`)
+
+      // Call a helper method that conducts the decrement and produces the output embed
+      return this._removeLitQuizPoint(userId, userRepository, tokenized[0])
+    })
+  }
+
+  // Helper method for the 'addpoint' dispatcher callback
+  // Returns an embed that displays the number of points of the target user after the increment
+  async _addLitQuizPoint (id, userRepository, userHandle) {
+
+    const result = await userRepository.incrementField(id, 'quiz_score')
+    if (result == null) {
+      throw new Error(`No existing record for user ${id}`)
+    }
+    return {
+      embeds: [{
+        title: MODULE_NAME,
+        description: `Added 1 point to ${userHandle} (now ${result.quiz_score}).`,
```

Comment:
Let's keep this consistent with quiz_adapter.js which bears the message "X now has Y point(s)."

Reply:
👌